include/index.h

```diff
@@ -272,47 +211,19 @@ namespace diskann {
     // determines navigating node of the graph by calculating medoid of data
     unsigned calculate_entry_point();
 
-    // called only when _eager_delete is to be supported
-    void update_in_graph();
-
     template<typename IDType>
     std::pair<uint32_t, uint32_t> search_impl(const T *query, const size_t K,
                                               const unsigned L, IDType *indices,
-                                              float *               distances,
-                                              InMemQueryScratch<T> &scratch);
+                                              float                *distances,
+                                              InMemQueryScratch<T> *scratch);
```

Comment:
Because scratch manager now returns a pointer to scratch space. This is to avoid unnecessary copies that we previously removed from SSD index scratch space